pos-print/src/main/java/com/clouway/pos/print/printer/FP705Printer.java

```diff
@@ -173,11 +167,18 @@ public PrintReceiptResponse printFiscalReceipt(Receipt receipt) throws IOExcepti
       channel.sendPacket(buildPacket(seq++, FISCAL_RECEIPT_PRINT_TEXT, params(prefix)));
     }
 
+
     double sum = 0;
     for (ReceiptItem item : receipt.getReceiptItems()) {
+      int vat = 1;
```

Comment:
vatGroup instead of just vat as it's not clear. 

As I remember it has to be string instead of int. 

String vatGroup = DEFAULT_VAT_GROUP; // will make things clear that default one will be used.

Reply:
fixed